concepts/basics/about.md

```diff
@@ -0,0 +1,92 @@
+# Haskell Syntax Basics
+
+## Constants and functions
+
+Haskell programs are composed of definitions and expressions.
+
+A constant value is defined with `name = expression`.
+
+```haskell 
+five = 5
+
+six = 3 + 3
+```
+
+Functions are defined with `name parameters... = expression`, parameters being only separated by space.
+
+```haskell
+add number1 number2 = number1 + number2
+```
+
+Invoking a function is also an expression and is done by specifying its name and passing arguments for each of the function's parameters,
+separated by a space, just like function definitions.
+
+```haskell
+five = add 2 3
+```
+
+Parentheses can be used to specify the order of evaluation of an expression.
+
+```haskell
+six = add 2 (1 * 4)
+
+twelve = add 2 1 * 4
+```
+
+## Indentation / significant whitespace
+
+Haskell uses whitespaces and indentation to specify code boundaries. 
+***Code which is part of some expression should be indented further than the beginning of that expression.***
+
+- Module, import, and top-level function definitions must start at the left most column.
+- If an expression is split into multiple lines, the code that is part of that expression must be indented under that expression with at least one space.
+- Parts of the expression that are grouped together should be indented with equal number of spaces.
+
+```haskell
+-- A function split over multiple lines, so subsequent lines must be indented
+add number1 number2 =
+    number1 + number2
+```
+https://en.m.wikibooks.org/wiki/Haskell/Indentation
```

Comment:
Do we expect more people to view this on a mobile device? I might suggest we link to the regular non-mobile https://en.wikibooks.org/wiki/Haskell/Indentation instead.

Reply:
Impressive. How did you catch that?